pkg/template/worker_template.go

```diff
@@ -351,6 +351,12 @@ storage:
       contents:
         source: "data:text/plain;charset=utf-8;base64,{{  index .Files "conf/10-docker.rules" }}"
 
+    - path: /etc/docker/daemon.json
+      filesystem: root
+      mode: 0600
+      contents:
+        source: "data:text/plain;charset=utf-8;base64,{{  index .Files "conf/docker-daemon.json" }}"
+
```

Comment:
Just for worker now. Should we add mirror for control planes as well?

Reply:
We need it for tenant cluster masters, sure. In case of a machine going down during an outage.

Reply:
Let's do all. Why not?